source/docs/software/kinematics-and-odometry/swerve-drive-odometry.rst

```diff
@@ -6,9 +6,17 @@ A user can use the swerve drive kinematics classes in order to perform :ref:`odo
 
 Creating the odometry object
 ----------------------------
-The ``SwerveDriveOdometry<int NumModules>`` class requires one template argument (only C++), two mandatory arguments, and one optional argument. The template argument (only C++) is an integer representing the number of swerve modules. The mandatory arguments are the kinematics object that represents your swerve drive (in the form of a ``SwerveDriveKinematics`` class) and the angle reported by your gyroscope (as a Rotation2d). The third optional argument is the starting pose of your robot on the field (as a ``Pose2d``). By default, the robot will start at ``x = 0, y = 0, theta = 0``.
+The ``SwerveDriveOdometry<int NumModules>`` class requires one template argument (only C++), three mandatory arguments, and one optional argument. The template argument (only C++) is an integer representing the number of swerve modules.
 
-.. note:: 0 degrees / radians represents the robot angle when the robot is facing directly toward your opponent's alliance station. As your robot turns to the left, your gyroscope angle should increase. By default, WPILib gyros exhibit the opposite behavior, so you should negate the gyro angle.
+The mandatory arguments are:
+
+* The kinematics object that represents your swerve drive (in the form of a ``SwerveDriveKinematics`` class)
+* The angle reported by your gyroscope (as a ``Rotation2d``)
+* The initial positions of the swerve modules (as an array of ``SwerveModulePosition``). In Java, this must be constructed with each wheel position in meters. In C++, the units library must be used to represent your wheel positions.
```

Comment:
Be consistent with the phrasing of how the parameter is passed. Applies to mecanum as well.